app/experimenter/nimbus-ui/src/components/ChangeApprovalOperations/RejectionReason.tsx

```diff
@@ -21,13 +18,13 @@ export const RejectionReason = ({
     rejectionEvent!;
 
   const rejectionActionDescription = useMemo(() => {
-    if (oldStatus === NimbusChangeLogOldStatus.LIVE) {
-      if (oldStatusNext === NimbusChangeLogOldStatusNext.LIVE) {
```

Comment:
Oh wait, I want to understand why we are removing `NimbusChangeLogOldStatus`?? 💭

Reply:
great explanation 🥳 👏 , now its clear, thank you @jaredlockhart 🙏